### ISSUE TITLE
Theme is preventing clicking on emotes

### ISSUE THREAD

**Issue by luxidoptera** [puckzxz/NotAnotherAnimeTheme#90]
Recently started having a problem with this theme wherein I can't click on or even hover over any emotes listed at the top of the emote list. It's as if an invisible, unclickable window is positioned overtop the emotes. You can see a GIF of the issue I'm having below- notice how none of the emotes I am hovering over show a highlight, or change the emote name at the top of the screen.

![naat glitch](https://user-images.githubusercontent.com/15697751/51623420-aa5d0b00-1f06-11e9-9e4a-67ee9a4516ca.gif)

Scrolling up and down in the list sometimes fixes this, sometimes does not. I have confirmed that this is an issue with this theme and not an issue with BetterDiscord or Discord itself, as disabling the theme causes this to stop happening.

For the record- I am running Bandaged BetterDiscord on Discord PTB, last refreshed just a couple minutes ago at the time of posting this issue. I also have a Nitro Classic subscription at the moment.

**Comment by puckzxz** [comment]
Alright, I have pushed out a fix for the issue in this commit https://github.com/puckzxz/NotAnotherAnimeTheme/commit/ba3cdb489d8e09d75244a3e96dcade6d0828ca6a

The issue was with certain elements being transparent in the emoji menu casuing emojis to be visible when they shouldn't.  I have removed the emoji menu transparency for now.  @Qu4k3 and I will look at safer ways to allow the users to customize their emoji menu.

I have not been able to recreate this issue after my fix using the Discord and Discord PTB client.

I'll leave this issue open for now in case you're still having problems.
